frontend/src/views/Settings/OrgSettingsPage/components/OrgAuthTab/LDAPModal.tsx

```diff
@@ -0,0 +1,229 @@
+import { useEffect } from "react";
+import { Controller, useForm } from "react-hook-form"; 
+import { yupResolver } from "@hookform/resolvers/yup";
+import * as yup from "yup";
```

Comment:
We are moving towards zod. To make the validation same as both frontend and backend. 

Can you please switch to zod and any new forms you have created

Reply:
Updated in latest version of PR that I will push soon.